prov/tcp/src/tcpx.h

```diff
@@ -184,9 +188,16 @@ enum tcpx_cm_state {
 	TCPX_EP_ERROR,
 };
 
+struct tcpx_rx_hdr {
+	struct ofi_op_hdr	hdr;
+	uint64_t		done_len;
```

Comment:
'done_len' should not be part of a header.  Is this really a wire structure? If not, I would rename 'hdr' to something else to avoid confusion.